src/calculateRentalCost.js

```diff
@@ -4,7 +4,23 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const baseCost = 40;
```

Comment:
rename all your constants to uper snake case, it's a good practice to use this naming for constants
`baseCost -> BASE_COST`